### PR TITLE
[STRAT-114] Some minor changes to the refactor for consistency, helpfulness

### PR DESCRIPTION
**Pull request by ufshk**
I've changed the names of functions/variables for consistency and also so that when we import the module anywhere there aren't any problems.

For elevations, there was some code that was for making requests, which Emma had asked me to move to a common code file (which was PR'd previously). I've removed that code so that the common file function is used in all cases.

Tests will be based on this PR.

### CODE REVIEW

File: routemodel/data_retrieval/routes.py [hunk 1]
```diff
@@ -55,14 +55,14 @@ def format_url_query(points: str, route_attrs = 'routePath', dist_unit = 'km'):
     @return: a string of formatted parameters to query from API
     """
     # adjust url for Route API request
-    url = 'Routes?'
+    query = ' Routes?'
```
Comment (e-wai) [inline routemodel/data_retrieval/routes.py:58]: Space at beginning?

**Review by e-wai — APPROVED** [review]
Everything looks good to me, but just check that space?

File: routemodel/data_retrieval/routes.py (hunk 1 above)
Comment (ufshk) [reply routemodel/data_retrieval/routes.py:58]: just imagine it doesn't exist

Comment (ufshk) [reply routemodel/data_retrieval/routes.py:58]: imagine

**Review by e-wai — DISMISSED** [review]

**Comment by e-wai** [comment]
No renaming?

**Review by mdshiozaki — DISMISSED** [review]
Probably should rename but i'm ok with it

**Comment by ufshk** [comment]
i keep making these pushes and dismissing reviews :(

**Comment by e-wai** [comment]
Still fine? We can continue to debate on the renaming `common.py` thing, but I'll approve anyways

**Review by e-wai — DISMISSED** [review]

**Review by e-wai — APPROVED** [review]
Sorry I missed it - still looks fine but probably merge conflicts waiting for you...

**Comment by ufshk** [comment]
I'll never forget you STRAT-114 PR #3.

**Pull request closed without merging**